eo-maven-plugin/src/main/java/org/eolang/maven/objectionary/OyRemote.java

```diff
@@ -71,6 +73,22 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    @Override
+    public boolean contains(final String name) throws MalformedURLException {
+        final URL url = this.template.value(name);
+        boolean presence;
+        try (InputStream inputStream = new Unchecked<>(() -> url).value().openStream()) {
+            presence = true;
+        } catch (final IOException ex) {
+            Logger.debug(
+                this, "The object '%s' is absent in %s...",
+                name, url
+            );
+            presence = false;
```

Comment:
@MikhailLipanin Please, read https://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#exceptions

Reply:
@volodya-lombrozo Here, I don't know another way. In our project there is also places, when we doing this: [here](https://github.com/objectionary/eo/blob/master/eo-maven-plugin/src/main/java/org/eolang/maven/util/Online.java#L64-L71),  [here](https://github.com/objectionary/eo/blob/master/eo-maven-plugin/src/test/java/org/eolang/maven/OnlineCondition.java#L44-L55). So may be here it is allowed?

Reply:
@MikhailLipanin I looks like you hide a problem here (`OyRemote`). Ok, let's create a puzzle for it.
As for `OyHome`  - we can remove catch for `InvalidPathException`

Reply:
added TODO in `OyRemote`, in `OyHome` Exception removed